etna/packages/etna-js/utils/fetch.js

```diff
@@ -1,3 +1,6 @@
+import downloadjs from 'downloadjs';
+import * as Cookies from './cookies';
+
 export const checkStatus = (response) => {
   let content = isJSON(response) ? response.json() : response.text();
```

Comment:
`checkStatus` used to be semi-duplicated. There was a version in Timur that returned the response object, and this version in `etna-js` that returned a Promise. I've consolidated to the Promise version and updated the consumers accordingly.